flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -1162,6 +1164,7 @@ public TableResult executeInternal(Operation operation) {
                 throw new TableException(exMsg, e);
             }
         } else if (operation instanceof ExplainOperation) {
+            // In fact, never get here
```

Comment:
It is reachable if the input is instance of `ExplainOperation`. 

Remove this comment